doc/reference/std/html.md

```diff
@@ -0,0 +1,188 @@
+# HTML Parser and Printer
+
+HTML is a widely used Markup Language that, while very similar to [XML](./xml.md), differs enough to have its own specific libraries.
+
+*Usage*
+
+```scheme
+(import :std/html)
+```
+
+This is for parsing and printing HTML. For HTML usage and templates for web development have a look at [The Template Attribute Language (TAL)](tal/README.md).
+
+```scheme
+(import ./html/parser)
+(export (import: ./html/parser))
+;;; This library is tangled from html/README.org
+```
+
+
+# Reading
+
+`html-parser` is intended as a permissive HTML parser for people who prefer the scalable interface described in Oleg Kiselyov's SSAX parser, as well as providing simple convenience utilities. It correctly handles all invalid HTML, inserting "virtual" starting and closing tags as needed to maintain the proper tree structure needed for the foldts down/up logic. A major goal of this parser is bug-for-bug compatibility with the way common web browsers parse HTML.
+
+
+## html->sxml
+
+```scheme
+(html->sxml
+ port-or-string
+ start: /default/ end: /default/ text: /default/ comment: /default/
```

Comment:
What is `/default/` ??

Reply:
default-html->sxml-plist. Says so next paragraph.